NEWS.md

```diff
@@ -8,6 +8,8 @@
 
 * fs is now licensed as MIT (#301).
 
+* `path_tidy()` now works with non-UTF8 encoded paths (#321).
```

Comment:
Thanks, can you add your GitHub username here as well?

Reply:
added